node/node.go

```diff
@@ -114,15 +114,19 @@ func New(cfg *Config, version string) (*Node, error) { //nolint:gocyclo,funlen
 	}
 
 	dbIsRemote := cfg.RemoteDB != ""
-	var database db.DB
-	if dbIsRemote {
-		database, err = remote.New(cfg.RemoteDB, context.TODO(), log, grpc.WithTransportCredentials(insecure.NewCredentials()))
-	} else {
-		database, err = pebble.New(cfg.DatabasePath, cfg.DBCacheSize, cfg.DBMaxHandles, dbLog)
+
+	createDB := func(dbPath string) (db.DB, error) {
+		if dbIsRemote {
+			return remote.New(dbPath, context.TODO(), log, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
I think there is an error instead of `dbPath` there should be `cfg.RemoteDB`

Reply:
also it's not clear why do we need this function

Reply:
I created this function previously when I used 2 databases. It's a minor refactor, but since we no longer use 2 dbs, I'll revert the changes.